$NetBSD: patch-ec,v 1.2 2007/05/12 13:53:11 ghen Exp $

ERROR: [check-portability.sh] => Found test ... == ...:
ERROR: [check-portability.sh] build/autoconf/mozconfig2configure: if [ "$APP" == "$MOZ_BUILD_APP" ]; then

--- build/autoconf/mozconfig2configure.orig	2006-02-22 17:19:52.000000000 +0100
+++ build/autoconf/mozconfig2configure
@@ -79,7 +79,7 @@ ac_add_options() {
 ac_add_app_options() {
   APP=$1
   shift;
-  if [ "$APP" == "$MOZ_BUILD_APP" ]; then
+  if [ "$APP" = "$MOZ_BUILD_APP" ]; then
       ac_add_options "$*";
   fi
 }
